slack/other channels

```diff
@@ -0,0 +1,13 @@
+Here are some of our other channels and why you might want to join them.
+
+#implementers - Those implementing the specification and questions about implementation details
+#specification - Potential changes to the JSON Schema specification
+#tests - Development of the official test suite
+#community - Development of the JSON Schema community and organisation
+#in-the-wild - Posting of any references to uses of JSON Schema by others for tracking
```

Comment:
This sentence feels awkward. I don't have a specific suggestion, but it might be worth having another look.

Reply:
"Find out and advertise where JSON Schema is used."  ?

Reply:
"References to observed uses of JSON Schemas in other applications or projects"